### DISCUSSION TITLE
Is there a way to get only the records of a user in the admin panel?

### DISCUSSION THREAD

**Discussion by Blanketville** [filamentphp/filament#1665] [Help]
Every user should have an admin panel with their own records.

**Comment by dmitrijsmihailovs** [comment]
You probably should look into multi tenancy. I think there was a discussion some time ago here about that.

**Reply by Blanketville** [reply]
That's a bit rich for my humble purposes.

**Comment by ryangjchandler** [comment]
@Blanketville You can modify the base query that is used on the resource, or perhaps use a global scope on the model?

**Reply by Blanketville** [reply]
Can't modify the base query on the resource, because $model takes a class name string of the model. A global scope on the model is an alternative, but with absolute restrictions.

**Reply by danharrin** [reply]
You can override `getEloquentQuery()` on the resource

**Reply by Blanketville** [reply]
Yes, this works like a dream, thanks.

**Reply by newtec-foggia** [reply]
how did you solve?

thanks

**Reply by danharrin** [reply]
https://filamentphp.com/docs/2.x/admin/resources/getting-started#simple-multi-tenancy-from-scratch
